sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/json/JsonSuite.scala

```diff
@@ -1083,83 +1083,59 @@ class JsonSuite extends QueryTest with SharedSQLContext with TestJsonData {
   }
 
   test("Corrupt records: PERMISSIVE mode, without designated column for malformed records") {
-    withTempView("jsonTable") {
-      val schema = StructType(
-        StructField("a", StringType, true) ::
-          StructField("b", StringType, true) ::
-          StructField("c", StringType, true) :: Nil)
+    val schema = StructType(
```

Comment:
While trying to check related other PRs, I saw some minor comments in https://github.com/apache/spark/pull/14929.

The actual changes are as below:

**From**

```
withTempView("jsonTable") {
  ...
  jsonDF.createOrReplaceTempView("jsonTable")
  ...
    sql(
      """
         |SELECT a, b, c
         |FROM jsonTable
      """.stripMargin),
  ...
}
```

**To**

```
jsonDF.select($"a", $"b", $"c"),
```